### PR TITLE
Anomaly events & science point gen tweaks

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

- added station event for anomaly spawning and made bluespace artifact event work with severity
- nerfed the shit out artifact point value. like holy shit what was i thinking. a single node could be close to 20000 points
- also nerfed anomaly value a bit. this should incentivize using both subdepartments, as well as expanding and getting more vessels.
- made anomalies burn you on touch and on hit. stop fucking hitting them you idiots.
- adds missing upgrade text for vessels
- makes anomaly stability randomly fluctuate on each pulse.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Reduced point output for both anomalies and artifacts
- tweak: Added a station event for anomalies to spawn.
- tweak: The superheated anomalous surface of an anomaly imparts nasty burns, even through protective covering.
- fix: The bluespace artifact station event now scales properly with survival rounds.
- fix: Fixed missing upgrade examine text on the anomaly vessel.
- add: Anomalies now slightly fluctuate in stability on each pulse. Better hope you don't push it over the edge.

### CODE REVIEW

File: Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactSystem.cs [hunk 1]
```diff
@@ -43,24 +40,28 @@ private void OnInit(EntityUid uid, ArtifactComponent component, MapInitEvent arg
...
-    /// Simple (1-2 Nodes): 1-2K
-    /// Medium (5-8 Nodes): 6-7K
-    /// Complex (7-12 Nodes): 10-11K
+    /// Simple (1-2 Nodes): ~10K
+    /// Medium (5-8 Nodes): ~30-40K
+    /// Complex (7-12 Nodes): ~60-80K
+    ///
+    /// Simple artifacts should be enough to unlock a few techs.
+    /// Medium should get you partway through a tree.
+    /// Complex should get you through a full tree and then some.
```
Comment (mirrorcult) [inline Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactSystem.cs:49]: this is for price not research points isnt it?

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
also im still personally of the opinion that damaging artifacts should reduce their health but out of scope

**Comment by EmoGarbage404** [comment]
> also im still personally of the opinion that damaging artifacts should reduce their health but out of scope

I plan on adding anomaly-specific methods of destabilization/reducing health. I feel like just reducing it to being able to beat the shit out of them to contain it will really reduce the threat that they can pose.

**Comment by mirrorcult** [comment]
er, i meant reducing stability

File: Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactSystem.cs (hunk 1 above)
Comment (EmoGarbage404) [reply Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactSystem.cs:49]: yep. my bad

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**